### PR TITLE
Add zones to /me

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

**Review by OverGlass — APPROVED** [review]
LGTM

**Review by OverGlass — CHANGES_REQUESTED** [review]
Les tests ne passent pas

**Review by OverGlass — APPROVED** [review]
LGTM

File: src/Normalizer/AdherentNormalizer.php [hunk 1]
```diff
@@ -17,7 +23,14 @@ public function __construct(
...
         private readonly CampaignHistoryRepository $campaignHistoryRepository,
         private readonly AdherentChangeEmailTokenRepository $changeEmailTokenRepository,
+        private readonly AdherentInstances $adherentInstances,
+        private readonly DataSurveyRepository $dataSurveyRepository,
+        private readonly AuthorizationCheckerInterface $authorizationChecker,
+        private readonly ClientRepository $clientRepository,
     ) {
+        if (!$adherentInstances) {
+            throw new \LogicException('AdherentInstances service is not initialized.');
+        }
```
Comment (ottaviano) [inline src/Normalizer/AdherentNormalizer.php:33]:
```suggestion
```

File: features/api/user.feature [hunk 2]
```diff
@@ -68,7 +84,26 @@ Feature:
...
+                "cadre_auth_path": "/oauth/v2/auth?scope=jemengage_admin&response_type=code&client_id=4498e44f-f214-110d-8b76-98a83f9d2b0c&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fauth",
+                "instances": {
+                    "assembly": {
+                        "type": "assembly",
+                        "name": "Paris (75)"
+                    },
+                    "circonscription": {
+                        "type": "circonscription",
+                        "name": "1ère circonscription • Paris (75-1)"
+                    },
```
Comment (ottaviano) [inline features/api/user.feature:96]: Could you add the code property for these two levels, please ?

**Review by ottaviano — APPROVED** [review]

**Pull request merged**